src/services/cesium/cesium.service.ts

```diff
@@ -10,6 +10,7 @@ export class CesiumService {
   private cesiumViewer: any;
 
   constructor(private ngZone: NgZone, private viewerFactory: ViewerFactory, @Optional() private viewerConfiguration: ViewerConfiguration) {
+    Cesium.BingMapsApi.defaultKey = 'AtIXLwI-EOgRtuDna0r-bcIAfj7G_cN6fb98u3A1DbgEEW-SquLhxEi8KnGrlJkA';
```

Comment:
remove it from cesium service,
add it to the demo or where it could be configured